lib/index.js

```diff
@@ -128,15 +128,19 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
       // make sure session data includes whether or not token client has
       // registered
       if(!authenticatedMethods.includes('token-client-registration')) {
-        // FIXME: get client ID from cookie
-        const clientId = 'FIXME';
-        const result = await brAuthnToken.clients.isRegistered(
-          {actor: null, email, account, clientId});
-        if(result.registered) {
-          authenticatedMethods.push('token-client-registration');
+        // get clientId from cookie
+        const clientId = req.cookies.cid;
+        // if clientId exists, check if client has been registered. If it is
+        // registered, then add `token-client-registration` to the list of
+        // `authenticatedMethods`.
```

Comment:
I think remove this, don't think this comment adds any value in addition to the simple code.

Reply:
Updated.